src/controllers/softtakeover.h

```diff
@@ -13,13 +13,11 @@
 
 class ControlObject;
 
-// 3/128 units away from the current is enough to catch fast non-sequential moves
-//  but not cause an audibly noticeable jump.
-
-
 class SoftTakeover {
   public:
-    static const double kDefaultTakeoverThreshold;
+    // 3/128 units away from the current is enough to catch fast non-sequential moves
+    //  but not cause an audibly noticeable jump.
+    static const double kDefaultTakeoverThreshold = 3.0 / 128;
```

Comment:
Either way it's a feature that was introduced in c++11 -- your compilers are being nice and not failing because they already have the feature implemented and you didn't specifically build with ```-std=c++03``` :)